.github/workflows/build-and-test.yml

```diff
@@ -222,6 +222,7 @@ jobs:
       - name: Gen genotelcontribcol
         run: |
           make genotelcontribcol
+          git diff
```

Comment:
debug?

Reply:
yes, it's probably not hurting to leave it

Reply:
ah ok I'll remove it

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33209